app/indexers/hyrax/valkyrie_work_indexer.rb

```diff
@@ -18,6 +18,7 @@ def to_solr # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Met
         admin_set_label = admin_set_label(resource)
         solr_doc['admin_set_sim']   = admin_set_label
         solr_doc['admin_set_tesim'] = admin_set_label
+        solr_doc['isPartOf_ssim'] = [resource.admin_set_id.to_s]
```

Comment:
Since you are adding this, can you update the hardcoded use of 'isPartOf_ssim' to use the configured relationship predicate?